python/hail/docs/tutorials/hail-overview.ipynb

```diff
@@ -777,18 +717,16 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Variant QC is a bit more of the same: we can use the [row_schema](https://hail.is/docs/devel/hail.MatrixTable.html#hail.MatrixTable.row_schema) method to produce a variety of useful statistics, plot them, and filter."
+    "Variant QC is a bit more of the same: we can use the [variant_qc](https://hail.is/docs/devel/methods/genetics.html?highlight=variant%20qc#hail.methods.variant_qc) method to produce a variety of useful statistics, plot them, and filter."
```

Comment:
The hyperlink isn't showing up. I'm not sure why. I can look into this next week.